src/tokens/_z-index.ts

```diff
@@ -0,0 +1,17 @@
+import type {TokenGroup} from './tokens';
+
+export const zIndex: TokenGroup = {
+  'z-content': '100',
+  'z-overlay': '400',
+  'z-globalRibbon': '510',
+  'z-top-bar': '512',
+  'z-context-bar': '513',
+  'z-small-screen-loading-bar': '514',
+  'z-nav-backdrop': '515',
+  'z-nav': '516',
+  'z-skip-to-content': '517',
+  'z-backdrop': '518',
+  'z-modal': '519',
+  'z-toast': '520',
+  'z-devUi': '521',
```

Comment:
I am interested to know what people think about keeping these semantic names? I think they offer little value and make it slower to understand what sits on top of other elements. I think we could radically simplify it to `z-521` for now.

Reply:
I agree that the semantic names don't add value and should be removed. Either of those suggested naming conventions feel like they could work as a placeholder until we refine/rename them in our second pass over the tokens.